src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[new Random().nextInt(1000)];
```

Comment:
1000 is a magic number - let's create a constant field (public static final int MAX_FIGURE_COUNT = 1000;)